### ISSUE TITLE
Remove line regarding moving the catalog into the repository from the README

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse#3202]
## Description

These lines can be removed (or maybe rephrased to denote that we do not have our infrastructure declaration in the monorepo):

https://github.com/WordPress/openverse/blob/aa3a7f5b37f5baf590967b950aade0c18299e3cf/README.md?plain=1#L24-L25

The catalog has been added to the monorepo so this is no longer accurate.

**Comment by AlajeBash** [comment]
Hi i will like to contribute to this issue by rephrasing the statements on line 24 and 25 to `"This repository contains most of the codebase, including the Openverse catalog, which has now been integrated into the monorepo."` . Please assign me Thanks

**Comment by dhruvkb** [comment]
@AlajeBash it would be better to drop that paragraph entirely as it's not wise to have time-specific information in the `README.md`. Assigning it to you.

**Comment by dhruvkb** [comment]
Thanks @lucasferrazlima, I'll reassign it to you.
